e2e/__snapshots__/extendConfig.test.ts.snap

```diff
@@ -619,6 +616,12 @@ Object {
       "fabloConfig": "<absolute path>",
     },
     "tls": false,
+    "tools": Object {
+      "explorer": undefined,
```

Comment:
ja bym proponował, albo explicite dać `false`, albo pominąć to pole

Reply:
po tej zmianie https://github.com/softwaremill/fablo/pull/254#discussion_r770330779 naprawiło się

Reply:
teraz się drukuje `"tools": Object {}`